packages/DropdownMenu/src/index.tsx

```diff
@@ -1,31 +1,58 @@
 import React from 'react'
 import * as Ariakit from '@ariakit/react'
 import { CreateWuiProps, forwardRef, WuiProps } from '@welcome-ui/system'
+import { useTheme } from '@xstyled/styled-components'
 
 import { Arrow } from './Arrow'
 import { Item } from './Item'
 import { Separator } from './Separator'
 import * as S from './styles'
 
-export interface DropdownMenuOptions extends Ariakit.MenuProps {
+export interface DropdownMenuOptions extends Omit<Ariakit.MenuProps, 'gutter'> {
   /** add custom props from styled system on DropdownMenu inner */
   innerProps?: WuiProps
+  gutter?:
+    | 'xxs'
+    | 'xs'
+    | 'sm'
+    | 'md'
+    | 'lg'
+    | 'xl'
+    | 'xxl'
+    | '3xl'
+    | '4xl'
+    | '5xl'
+    | '6xl'
+    | '7xl'
```

Comment:
Get the space theme types instead or rewrite it

Reply:
I tried to get the theme with `ThemeSpace` from `@xstyled/styled-components` but its type is `string | number | true | symbol | {} ` rather than this list, so I had to add `@welcome-ui/core` to get it.